<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifierespacesDialog</class>
 <widget class="QDialog" name="modifierespacesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1501</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1451</width>
     <height>871</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(70, 126, 171);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>330</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */
qproperty-iconSize: 42px 42px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/Desktop/img/ecommerce_home_market_mart_shop_shopping_store_icon_123207.png</normaloff>:/Desktop/img/ecommerce_home_market_mart_shop_shopping_store_icon_123207.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>580</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */
qproperty-iconSize: 42px 42px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/Desktop/img/sidebar_utilities_18421.png</normaloff>:/Desktop/img/sidebar_utilities_18421.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>500</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */
qproperty-iconSize: 42px 42px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/Desktop/img/website_maintenance_settings_window_alert_icon_193897 (1).png</normaloff>:/Desktop/img/website_maintenance_settings_window_alert_icon_193897 (1).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>670</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */
qproperty-iconSize: 42px 42px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/Desktop/img/investment_asset_management_worker_employee_welfare_icon_210504.png</normaloff>:/Desktop/img/investment_asset_management_worker_employee_welfare_icon_210504.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>0</y>
      <width>271</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none; /* Supprime le fond */
qproperty-iconSize: 80px 80px; /* Augmente la taille de l'icône */
border: none; /* Supprime la bordure si nécessaire */

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/Desktop/img/logo_final-removebg-preview.ico</normaloff>:/Desktop/img/logo_final-removebg-preview.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>260</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */
qproperty-iconSize: 42px 42px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/Desktop/img/validated_buyer_user_apply_done_icon_177174 (1).png</normaloff>:/Desktop/img/validated_buyer_user_apply_done_icon_177174 (1).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>420</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */
qproperty-iconSize: 42px 42px; /* Ajuste la taille de l'icône */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/Desktop/img/calendar_date_day_event_month_schedule_time_icon_123230.png</normaloff>:/Desktop/img/calendar_date_day_event_month_schedule_time_icon_123230.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="ajout">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>120</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
font: 700 9pt &quot;Segoe UI&quot;;
border: 2px solid black ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
qproperty-iconSize: 30px 30px; /* Ajuste la taille de l'icône */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */</string>
    </property>
    <property name="text">
     <string>ajouter </string>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/Desktop/img/18926930-conception-d-icone-de-formulaire-gratuit-vectoriel-removebg-preview.png</normaloff>:/Desktop/img/18926930-conception-d-icone-de-formulaire-gratuit-vectoriel-removebg-preview.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="liste">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>120</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
font: 700 9pt &quot;Segoe UI&quot;;
border: 2px solid black ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
qproperty-iconSize: 30px 30px; /* Ajuste la taille de l'icône */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */</string>
    </property>
    <property name="text">
     <string>liste</string>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/Desktop/img/570170-removebg-preview.png</normaloff>:/Desktop/img/570170-removebg-preview.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="stat">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>120</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
font: 700 9pt &quot;Segoe UI&quot;;
border: 2px solid black ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
qproperty-iconSize: 30px 30px; /* Ajuste la taille de l'icône */
border-radius: 30px; /* Ajuste selon le degré d'arrondi souhaité */</string>
    </property>
    <property name="text">
     <string>statistique</string>
    </property>
    <property name="icon">
     <iconset resource="IMG.qrc">
      <normaloff>:/Desktop/img/1170667-removebg-preview.png</normaloff>:/Desktop/img/1170667-removebg-preview.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 25px; /* Ajuste selon le degré d'arrondi souhaité */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::EditFind"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>1221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 25px; /* Ajuste selon le degré d'arrondi souhaité */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>581</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : none ; 
font: 700 22pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>                        S.M.M</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
font: 14pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string>-Smart Mall Manger-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SystemLogOut"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>40</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : none ; 
font: 700 18pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>gestion des espaces</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>1191</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
border: 2px solid; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 25px; /* Ajuste selon le degré d'arrondi souhaité */
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(227, 241, 244);
font: 900 12pt &quot;Segoe UI&quot;;
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 10px; /* Ajuste selon le degré d'arrondi souhaité */
</string>
     </property>
     <property name="text">
      <string>Retour </string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>480</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 12pt &quot;Segoe UI&quot;;
qproperty-iconSize: 14px 14px; /* Ajuste la taille de l'icône */
border-radius: 10px; /* Ajuste selon le degré d'arrondi souhaité */
</string>
     </property>
     <property name="text">
      <string>Quitter </string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::SystemLogOut"/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>701</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(176, 186, 196);
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nom_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
      </property>
      <property name="text">
       <string>Type </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
      </property>
      <property name="text">
       <string>Superficie  </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
      </property>
      <property name="text">
       <string>Date_Debut_Contrat </string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
      </property>
      <property name="text">
       <string>Emplacement </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Emplacement_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
      </property>
      <property name="text">
       <string>Date_fin_contrat</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>300</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
      </property>
      <property name="text">
       <string>Statut </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Type_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_Superficie_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;
</string>
      </property>
      <property name="text">
       <string>Loyer_Mensuel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_loyer_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>220</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>175</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 241, 244);
border: 12px solid transparent ; /* Ajoute une bordure */
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 15px; /* Ajuste selon le degré d'arrondi souhaité */

</string>
      </property>
      <item>
       <property name="text">
        <string>libre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>occupé</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="enregistrer">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>100</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 4px solid black;
background-color: rgb(132, 194, 49);
border-radius: 20px;
font: 700 12pt &quot;Arial Rounded MT&quot;;</string>
      </property>
      <property name="text">
       <string>enregistrer </string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>561</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 35pt &quot;Berlin Sans FB&quot;;
font: 700 35pt &quot;Urdu Typesetting&quot;;
border: none; /* Supprime toute bordure */


</string>
     </property>
     <property name="text">
      <string>Gestion des espaces</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="IMG.qrc"/>
 </resources>
 <connections/>
</ui>
